stripe/src/main/java/com/stripe/android/net/StripeApiHandler.java

```diff
@@ -135,9 +137,72 @@ public static Source createSourceOnServer(
             InvalidRequestException,
             APIConnectionException,
             APIException {
+        return createSourceOnServer(
+                uidProvider,
+                context,
+                sourceParams,
+                publishableKey,
+                null,
+                loggingResponseListener);
+    }
+
+    /**
+     * Create a {@link Source} using the input {@link SourceParams}.
+     *
+     * @param uidProvider a provider for UUID items in test
+     * @param context a {@link Context} object for aquiring resources
+     * @param sourceParams a {@link SourceParams} object with {@link Source} creation params
+     * @param publishableKey an API key
+     * @param stripeAccount a connected Stripe Account ID
+     * @param loggingResponseListener a listener for logging responses
+     * @return a {@link Source} if one could be created from the input params,
+     * or {@code null} if not
+     * @throws AuthenticationException if there is a problem authenticating to the Stripe API
+     * @throws InvalidRequestException if one or more of the parameters is incorrect
+     * @throws APIConnectionException if there is a problem connecting to the Stripe API
+     * @throws APIException for unknown Stripe API errors. These should be rare.
+     */
+    @Nullable
+    public static Source createSourceOnServer(
```

Comment:
The Stripe Account is set on the Stripe object, just one time. I had to feed it to the StripeApiHandler object in this way only for sources because RequestOptions aren't fed to this particular method (they're an argument in the createToken methods). All stripe api calls have the Stripe-Account header attached to them once you set it on a particular Stripe object